views/partials/searchCriteria.ejs

```diff
@@ -18,5 +18,12 @@
             <input type="checkbox" value="3">
         </div>
     </div>
+    <select name="word-limit" id="word-limit">
+        <option value="1">1</option>
+        <option value="5">5</option>
+        <option value="10">10</option>
+        <option value="25">25</option>
+        <option value="50">50</option>
+    </select>
     <button id="search-button" class="search-button" type="submit"> Search! </button>
```

Comment:
Same comment about identical id and class names. If you're using the id as a handler for an event, let's say - thus in JavaScript, you can name it accordingly.